### ISSUE TITLE
Exception when selecting Realism Overhaul for update.

### ISSUE THREAD

**Issue by bmihai** [KSP-CKAN/CKAN#1629]
Hello,

**CKAN Version:**  1.16.1-0-g2e91715
**Operating System:** Windows 7
**The issue you are experiencing:** CKAN crashes with the message below when selecting Realism Overhaul for update(from 10.9.3 to 10.9.5). CrossFeedEnabler is already installed, but shows up as compatible with KSP 1.0.4 only (1.0.5 is installed).
**How to recreate this issue:** Click on the "Update" checkbox next to Realism Overhaul.
**CKAN error codes (if applicable):** 
************** Exception Text **************
CKAN.ModuleNotFoundKraken: Cannot install CrossFeedEnabler, module not available
   at CKAN.CkanModule.FromIDandVersion(IRegistryQuerier registry, String mod, KSPVersion ksp_version)
   at CKAN.RelationshipResolver.<RelationshipResolver>c__AnonStorey0.<>m__0(String name)
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at CKAN.RelationshipResolver..ctor(IEnumerable`1 module_names, RelationshipResolverOptions options, IRegistryQuerier registry, KSPVersion kspversion)
   at CKAN.MainModList.ComputeConflictsFromModList(IRegistryQuerier registry, IEnumerable`1 change_set, KSPVersion ksp_version)
   at CKAN.Main.<UpdateChangeSetAndConflicts>c__async1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at CKAN.Main.<ModList_CellValueChanged>c__async0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)

**Comment by politas** [comment]
CrossFeedEnabler has been removed as a dependency from RealFuels, which I believe is the only dependency inheritance for it. It is no longer needed, and should be removed. Try uninstalling it.

**Comment by bmihai** [comment]
You're right! Uninstalling **CrossFeedEnabler** stopped the exception and allowed me to do the upgrade. The error message was a bit confusing, because it seemed to complain about it missing (it probably meant it's an unsuitable version).

I think I must have installed it from the optionals list, I'm not sure...

I got a similar error when trying to remove the now-deprecated **TestFlight Config Realism Overhaul**. I had to uninstall **TestFlight** itself too, and then put it back on its own.

Thank you for looking into this!
